File: sjokolade.py
"""
Gå gjennom tallene 1-100

Hvis tallet kan deles på 3, print Sjoko
Hvis tallet kan deles på 5, print Lade
Hvis tallet kan deles på 3 og 5, print SjokoLade
Ellers print tallet (i)
"""

for i in range(0, 101):
        if i % 3 == 0 and i % 5 == 0:
            print(i, ': Sjokolade')
        elif i % 3 == 0:
            print(i, ': Sjoko')
        elif i % 5 == 0:
            print(i, ': lade')
        else:
            print(i, (f': {i}'))

File: Sjokolade.js
/*I denne oppgaven skal du gå gjennom tallene 1-100

 Hvis tallet kan deles på 3, print sjoko til konsollen
 Hvis tallet kan deles på 5, print lade til konsollen
 Hvis tallet kan deles på både 3 og 5, print sjokolade til konsollen

 Lykke til :)
 */

 for( let i = 1; i <= 100; i++){
   if (i % 3 == 0 && i % 5 == 0){
      console.log(`${i} :sjokolade`)
   } else if (i % 3 == 0) {
      console.log(`${i} : sjoko`)
   } else if (i % 5 == 0) {
      console.log(`${i} : lade`)
   } else {
      console.log(`${i} : ${i}`)
   }
}

